### PR TITLE
migrate to v3 for charge data

### PR DESCRIPTION
**Pull request by drc38**

### CODE REVIEW

File: custom_components/evnex/sensor.py [hunk 1]
```diff
@@ -252,25 +292,31 @@ async def async_setup_entry(
 
     # Create an Evnex Org Sensor showing org wide weekly summary of powerUsage, charging sessions, cost
     entities.append(EvnexOrgWidePowerUsageSensorToday(coordinator=coordinator))
-    entities.append(EvnexOrgWideChargeSessionsCountSensor(coordinator=coordinator))
+    entities.append(EvnexOrgWideChargeSessionsCountSensor(
+        coordinator=coordinator))
```
Comment (hardbyte) [inline custom_components/evnex/sensor.py:296]:
```suggestion
    entities.append(EvnexOrgWideChargeSessionsCountSensor(coordinator=coordinator))
    entities.append(EvnexOrgTierSensor(coordinator=coordinator))
```

File: custom_components/evnex/sensor.py [hunk 2]
```diff
@@ -170,7 +188,8 @@ class EvnexChargePortConnectorStatusSensor(EvnexChargePointConnectorEntity, Sens
 
     @property
     def native_value(self):
-        brief = self.coordinator.data['connector_brief'][(self.charger_id, self.connector_id)]
+        brief = self.coordinator.data['connector_brief'][(
+            self.charger_id, self.connector_id)]
         return brief.status
```
Comment (hardbyte) [inline custom_components/evnex/sensor.py:193]:
```suggestion
        brief = self.coordinator.data['connector_brief'][(
            self.charger_id, self.connector_id)]
        return brief.ocppStatus
```

**Review by hardbyte — APPROVED** [review]
Just a couple of suggested tweaks then looks good to me

**Comment by hardbyte** [comment]
I've added you as a collaborator @drc38 - feel free to merge any approved PRs, make beta releases

File: custom_components/evnex/sensor.py (hunk 1 above)
Comment (hardbyte) [reply custom_components/evnex/sensor.py:296]: I'm not overly familiar with HA, but we can also put static data in the device info - I imagine the `OrgTier` doesn't ever change (right?). So perhaps should go in the org's device info?

**Comment by drc38** [comment]
Cheers, are the device attributes fixed (https://developers.home-assistant.io/docs/device_registry_index/#device-properties) or can  they be added to? If they can be customised then it probably does make sense to shift it to the device registry

**Comment by hardbyte** [comment]
As far as I can tell they are fixed. So it would be a bit of a kludge putting org tier as the "model". Let's go with your approach of adding it as another sensor.

**Pull request merged**